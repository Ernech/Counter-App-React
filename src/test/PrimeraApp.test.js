import '@testing-library/jest-dom';
import React from 'react';
import PrimeraApp from '../PrimeraApp';
import { shallow } from 'enzyme';

describe('Test at <PrimeraApp component/>', () => {
            // test('must show msg "Hello I am Mario" ', () => {
            //         const saludo = "Hello I am Mario";

            //         const { getByText } = render( < PrimeraApp saludo = { saludo }
            //             / > );

            //             expect(getByText(saludo)).toBeInTheDocument();
            //         });

            test('should show "<PrimeraApp/> correctly" ', () => {
                    const saludo = "Hello I am Mario";

                    const wrapper = shallow( < PrimeraApp saludo = { saludo }
                        / > );
                        expect(wrapper).toMatchSnapshot();
                    });

                test('should return a subtible send by props', () => {
                        const saludo = "Hello I am Mario";
                        const subtitulo = "and i am a subtitle";
                        const wrapper = shallow( < PrimeraApp saludo = { saludo }
                            subtitulo = { subtitulo }
                            / > );

                            const textoParrafo = wrapper.find('p').text(); console.log(textoParrafo); expect(textoParrafo).toBe(subtitulo)
                        });


                });