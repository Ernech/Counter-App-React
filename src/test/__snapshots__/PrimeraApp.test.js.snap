// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test at <PrimeraApp component/> should show "<PrimeraApp/> correctly"  1`] = `
<Fragment>
  <h1>
    Hello I am Mario
  </h1>
  <p>
    I am a subtitle
  </p>
</Fragment>
`;
